lib/pac_conn

```diff
@@ -1090,9 +1120,18 @@ if (defined $METHOD) {
             ctrl("PASSWORD:Asking user for password...");
             $PASS = wEnterValue(undef, '<b>Manual Password requested</b>', "Enter Password for '$NAME'", '', 0) // '';
         }
-
+        my $method = 'VNC';
         $CONNECT_OPTS =~ s/\s*-embed//go;
-        if (`vncviewer --help 2>&1 | /bin/grep TigerVNC`) {
+        if (`vncviewer --help 2>&1 | /bin/grep RealVNC`) {
+            $method = 'RealVNC';
+        } elsif (`vncviewer --help 2>&1 | /bin/grep TigerVNC`) {
+            $method = 'TigerVNC';
+        }
+        _setSshProxy($method);
+        if ($proxy_jump) {
```

Comment:
This should also happen for RDP (or any other method...).

Shouldn't we do this inside _setSshProxy ?

Reply:
Yes, has been moved and tested on my last commit

Reply:
I have been testing and I have this strange behaviour, where the exit signal has been sent to the ssh tunnel, but in some occasions is ignored and the process will not actually close. The trace shows that the signal is been sent and the socket closed but the process remains in memory.

Reply:
Found the problem, we have to close the tunnel before any hard close instruction is executed. Actually the signal was not sent to the tunnel session because of this.

Created a generic function that is called before any hard close, so we can control any other things that would have to be handled before a hard close.